fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -251,11 +251,15 @@ def copy_logfile(device, log_identity, logs_destination_dir)
 
       def copy_logarchive(device, log_identity, logs_destination_dir)
         require 'shellwords'
-
         logarchive_dst = Shellwords.escape(File.join(logs_destination_dir, "system_logs-#{log_identity}.logarchive"))
         FileUtils.rm_rf(logarchive_dst)
-        command = "xcrun simctl spawn #{device.udid} log collect --output #{logarchive_dst} 2>/dev/null"
-        FastlaneCore::CommandExecutor.execute(command: command, print_all: false, print_command: true)
+  #      FileUtils.mkdir_p(logs_destination_dir)
+        command = "xcrun simctl spawn #{device.udid} log collect --output #{logarchive_dst}"
+        begin
+          FastlaneCore::CommandExecutor.execute(command: command, print_all: true, print_command: true)
+        rescue FastlaneCore::Interface::FastlaneError
+          UI.error "Failed to copy simulator logs"
```

Comment:
i'm seeing an issue whereby in some cases simctl fails to copy a nonexistent diagnostics dir, but still copies the rest of the archive.

maybe @russbishop knows more?